### ISSUE TITLE
IP of connection name and peer_host seems wrong

### ISSUE THREAD

**Issue by Stefku** [rabbitmq/rabbitmq-server#1430]
* RabbitMQ version: 3.6.10
* Erlang version: "Erlang/OTP 19 [erts-8.2.1] [source] [64-bit] [smp:48:48] [async-threads:768] [hipe] [kernel-poll:true]"
* Operating system, version, and patch level: Linux 3.10.0-693.2.2.el7.x86_64 x86_64

I wonder why IP address seems wrong in connections. In HTTP API "/api/connections" there are name and peer_host of the connection. Always, the last number of IP address of peer_host is shown as 1. Even, if the corresponding server has a non-1 number there.

Example
```
"name": "10.1.1.1:37306 -> 10.1.25.89:5672",
"peer_host": "10.1.1.1",
"host": "10.1.25.89",
```

This is very confusing and makes it hard to reason about host of a connection.

**Comment by michaelklishin** [comment]
Thank you for your time.

Team RabbitMQ uses GitHub issues for specific actionable items engineers can work on. This assumes two things:

1. GitHub issues are not used for questions, investigations, root cause analysis, discussions of potential issues, etc (as defined by this team)
2. We have a certain amount of information to work with

We get at least a dozen of questions through various venues every single day, often quite light on details.
At that rate GitHub issues can very quickly turn into a something impossible to navigate and make sense of even for our team. Because of that questions, investigations, root cause analysis, discussions of potential features are all considered to be mailing list material by our team. Please post this to [rabbitmq-users](https://groups.google.com/forum/#!forum/rabbitmq-users).

Getting all the details necessary to reproduce an issue, make a conclusion or even form a hypothesis about what's happening can take a fair amount of time. Our team is multiple orders of magnitude smaller than the RabbitMQ community. Please help others help you by providing a way to reproduce the behavior you're
observing, or at least sharing as much relevant information as possible [on the list](https://groups.google.com/forum/#!forum/rabbitmq-users):

 * Server, client library and plugin (if applicable) versions used
 * Server logs
 * A code example or terminal transcript that can be used to reproduce
 * **Full** exception stack traces (not a single line message)
 * `rabbitmqctl status` (and, if possible, `rabbitmqctl environment` output)
 * Other relevant things about the environment and workload, e.g. [a traffic capture](https://www.rabbitmq.com/amqp-wireshark.html)

Feel free to edit out hostnames and other potentially sensitive information.

When/if we have enough details and evidence we'd be happy to file a new issue.

Thank you.

**Comment by michaelklishin** [comment]
The address is taken from the socket. I can think of two things that may be going on:

 * Your server has multiple interfaces, you inspect one but RabbitMQ/client connections use the other
 * Connections go through a load balancer and [proxy protocol support]() is not available in 3.6.x

See [Troubleshooting Networking](http://www.rabbitmq.com/troubleshooting-networking.html) for some starting points.
